tests/e2e/reconcile/01-install.yaml

```diff
@@ -18,16 +18,3 @@ spec:
   storage:
     secret: minio-test
   storageSize: 200M
-  resources:
```

Comment:
why removed this?

Reply:
Because kuttl runs e2e tests in parallel, and if there are too many tempo containers running, each requesting a specific amount of CPU resources, the kubernetes scheduler will stop scheduling them (because the GitHub Actions VM only has 2 cores), pods will be in pending state and the e2e test will fail.